premailer/premailer.py

```diff
@@ -534,8 +548,13 @@ def _style_to_basic_html_attributes(self, element, style_content,
                 attributes['align'] = value.strip()
             elif key == 'vertical-align':
                 attributes['valign'] = value.strip()
-            elif key == 'background-color':
-                attributes['bgcolor'] = value.strip()
+            elif ((key == 'background-color')
+                  and ('transparent' not in value.lower())):
```

Comment:
I have never found a Boolean expression without brackets that I have ever trusted. Feel free to change it.